agent/agent.go

```diff
@@ -680,20 +679,21 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string,
 	switch detail.Status {
 	case evergreen.TaskSucceeded:
 		tc.logger.Task().Info("Task completed - SUCCESS.")
+		a.handleTimeoutAndOOM(ctx, tc, status)
```

Comment:
I has to go right before L681, as I stated in my comment (same as the other one).

Reply:
my bad, I misread your previous comment. 
fixed